lib/cli.coffee

```diff
@@ -25,12 +26,20 @@ installMycha = (testHelperPath, done) ->
   installer.install done
 
 
+loadConfig = (done) ->
```

Comment:
You could just export the function from config_loader.coffee. 
Given that it doesn't return errors, you could remove the async.waterfall overhead and the two helper methods without losing much readability:

```coffeescript
configLoader = require './config_loader'

configLoader process.cwd, (err, config) ->
  Mycha.run {argv: process.argv.slice(2), config, cwd: process.cwd()}, (err, exitCode) ->
    ...
```

Just an idea.